index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Jornada Trybe</title>
+</head>
+
+<body>
+  <!--2. Adicione uma barra superior com um título-->
+  <header id="cabecalho">
+    <h1 id="titulo">Jornada Trybe</h1>
+    <nav>
+      <ul>
+        <li>Aprendizado na Trybe</li>
+        <li>Contatos</li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <!--3. Adicione uma foto sua à página  -->
+    <div class="imagem listas">
+      <img id="minha_foto" src="minha_foto.jpeg" alt="minha_foto">
+    </div>
+    <!--4. Adicione uma lista de lições aprendidas à página-->
+    <section class="listas">
+      <h2>Liçoes Aprendidas</h2>
+      <ol id="licoes_aprendidas">
+        <li>O que é Unix e Bash.</li>
+        <li>Principais comandos utilizados no Bash</li>
+        <li>O que é Git</li>
+        <li>O que é GitHub</li>
+        <li>Como sincronizar o Git com o GitHub</li>
+        <li>Entendendo a internet</li>
+        <li>Introdução sobre o HTML</li>
+        <li>Introdução sobre o CSS</li>
+        <li>HTML Semântico</li>
+        <li>Início ao conhecimento das soft skills</li>
+      </ol>
+    </section>
+    <section class="listas">
+      <!--5. Crie uma lista de lições que ainda deseja aprender para a página-->
+      <ul id="licoes_a_aprender">
+        <h2>Liçoes a Aprender</h2>
+        <li>Aprofundamento do HTML</li>
+        <li>Aprofundamento do CSS</li>
+        <li>JavaScript</li>
+        <li>Técnicas para melhorar o acesso do usuário</li>
+        <li>Técnicas para aumentar o engajamento da página</li>
+        <li>Trabalhar o design dos sites de forma ficar mais agradável</li>
+        <li>Formas eficientes de resolver "bugs" no site</li>
+        <li>Entender que outras aplicações são importantes também ter conhecimento ao fazer uma página, por exemplo, programas de edição de imagem</li>
+        <li>Aplicação Front-end</li>
+        <li>Aplicação Back-end</li>
+      </ul>
+    </section>
+    <br>
+    <!--8. Crie um artigo sobre seu aprendizado-->
+    <article class="caixa lado-direito">
+      <section>
+        <h2>Meu aprendizado na Trybe</h2>
+        <p>A Trybe apresenta um jeito único de fazer a gente aprender, aqui nós não nos desenvolvemos somente para seremprogramadores, mas sim, tudo que um verdadeiro profissional precisa possuir, como resiliência, respeito epreocupação pelo próximo. Vale afirmar também, que a ela nos fornece um material incrível, bem detalhado e comum nível além do básico. Além disso, conta com uma equipe de funcionários sensacional,que prestam apoio para todos independente do problema.</p>
```

Comment:
@GabrielFMPinheiro acredito que tenha uns erros tipográficos neste item. 
```suggestion
        <p>A Trybe apresenta um jeito único de fazer a gente aprender, aqui nós não nos desenvolvemos somente para sermos programadores, mas sim, tudo que um verdadeiro profissional precisa possuir, como resiliência, respeito e preocupação pelo próximo. Vale afirmar também, que ela nos fornece um material incrível, bem detalhado e com um nível além do básico. Além disso, conta com uma equipe de colaboradores sensacional, que prestam apoio para todos independente do problema.</p>
```
Sugiro a troca de `funcionários` por `colaboradores`, por este ser um termo mais bem aceito (pelo menos é o que diz o pessoal de rh) rsrs

Reply:
Verdade, vou substituir a palavra, também acho melhor. Obrigado!